pkgs/pysodium/default.nix

```diff
@@ -1,4 +1,5 @@
-{ version, src, stdenv, buildPythonPackage, libsodium }:
+{ pkgs, version, src }:
```

Comment:
With native python libs one can easily run into python version incompatibilities, using the old approach allows to use different python versions to build at least when picked from an overlay.

was there a specific reason to opt for this new format?

Reply:
I wanted to make sure that all packages are built using the same nixpkgs/python version. I understand you point though. I guess you would simply keep the ``buildPythonPackage`` parameter ?
(Btw: [Here](https://github.com/ngi-nix/opaque-sphinx/pull/2/files#diff-206b9ce276ab5971a2489d75eb1b12999d4bf3843b7988cbe8d687cfde61dea0R134) the function is declared and used implicitly. I guess it's actually good to make that more explicit.)